docs/submit.rst

```diff
@@ -0,0 +1,35 @@
+Submission of benchmarks
+========================
+
+After all benchmark systems are generated, you can also use MDBenchmark to
+submit these to the queuing system on your HPC. We currently support submission
+to `Slurm`_, `SGE`_ and `LoadLeveler`_.
+
+Submitting all generated benchmarks
+-----------------------------------
+
+To submit all generated benchmarks that are recursively found starting in the
+current directory, use::
+
+  mdbenchmark submit
```

Comment:
since we don't have the user confirmation implemented yet it might be good to really highlight the fact that this works recursively and one might end up submitting a bunch of unwanted stuff by accident.